platform/viewer/src/components/UserPreferences/UserPreferences.js

```diff
@@ -25,7 +27,7 @@ const tabs = [
     name: 'Window Level',
     Component: WindowLevelPreferences,
     customProps: {},
-    hidden: true,
+    hidden: false,
```

Comment:
Where is this used?

Reply:
It was already there, its used to display or not the tab, either the component and the header tab.

Reply:
I've set false as default and remove it from configuration, as they were all false.